app/lib/Plugins/SearchEngine/Elastic8/FieldTypes/Intrinsic.php

```diff
@@ -193,7 +193,8 @@ public function getRewrittenTerm(Zend_Search_Lucene_Index_Term $term): Zend_Sear
 				);
 			} else {
 				return new Zend_Search_Lucene_Index_Term(
-					'"' . $raw_term . '"', $term->field
+					// TODO!!!! DO NOT MERGE!!!!!!!!
+					'"VPRS ' . $raw_term . '"', $term->field . '.'. $this->getDataTypeSuffix(static::SUFFIX_IDNO)
```

Comment:
Config option giving me weirdness, will investigate further, hardcoded only to facilitate other testing.

Reply:
Yup no worries I understand